web/client/utils/mapinfo/wfs.js

```diff
@@ -27,6 +27,21 @@ import assign from 'object-assign';
  * @return {object} an object with `request`, containing request params, `metadata` with some info about the layer and the request, and `url` to send the request to.
  */
 const buildRequest = (layer, { map = {}, point, currentLocale, params, maxItems = 10 } = {}, infoFormat, viewer, featureInfo) => {
+    const { features = [] } = point?.intersectedFeatures?.find(({ id }) => id === layer.id) || {};
+    if (features.length) {
```

Comment:
here we should check if point?.intersectedFeatures is available instead of features.length because even an empty features array is a valid response.
intersectedFeatures represents the property that tell us if we should make or not the WFS requests.

```suggestion
    if (point?.intersectedFeatures) {
```